requirements/base.txt

```diff
@@ -6,3 +6,5 @@
 #
 django-object-actions==0.10.0
 django-waffle==0.12.0
+
+edx-opaque-keys==0.4.0
```

Comment:
I think you should add this in base.in and then use `make upgrade` to create `.txt` files